tests/docker/ducker-ak

```diff
@@ -399,7 +403,12 @@ attempting to start new ones."
     if docker network inspect ducknet &>/dev/null; then
         must_do -v docker network rm ducknet
     fi
-    must_do -v docker network create ducknet
+    network_create_args=""
+    if [[ "${ipv6}" == "true" ]]; then
+        subnet_cidr_prefix="${DUCKER_SUBNET_CIDR:-"fc00:cf17"}"
```

Comment:
I'm not knowledgeable about IPv6, so I'm wondering if this is a safe thing to hardcode?